cdap-common/src/main/java/co/cask/cdap/common/discovery/RandomEndpointStrategy.java

```diff
@@ -20,12 +20,12 @@
 
 import java.util.Iterator;
 import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
 
 /**
  * Randomly picks endpoint from the list of available endpoints.
  */
 public final class RandomEndpointStrategy extends AbstractEndpointStrategy {
-
```

Comment:
You shouldn't remove this line. Always have an empty line between class/methods.